src/project/internal/projectactionscontroller.cpp

```diff
@@ -508,7 +535,7 @@ void ProjectActionsController::saveProjectToCloud(const SaveLocation::CloudInfo&
     m_uploadingProgress = uploadingService()->uploadScore(*projectData, meta.title, meta.source);
 
     m_uploadingProgress->started.onNotify(this, [this]() {
-        m_saveToCloudAllowed = false;
+        m_isUploadingProject = false;
```

Comment:
You need to change the behavior of this flag after renaming it

Reply:
🤦 
Fixed!